### ISSUE TITLE
Property error: : resources.private_subnet.properties.dns_nameservers  is not a list

### ISSUE THREAD

**Issue by 10186937** [pndaproject/pnda-heat-templates#3]
Hi,
I get the next error, when I run the command to create a pnda cluster.
What's wrong? 
Thanks!

```
[root@devstack162 cli]# ./heat_cli.py create -e squirrel-land -f standard -n 3 -o 1 -k 1 -z 1 -s pnda -y
+---------+
| P N D A |
+---------+

Please wait while your PNDA cluster is being created.

This process can last for 1 or 2 hours.

openstack stack create --timeout 120 --wait --template pnda_standard.yaml --environment pnda_env_standard.yaml --parameter ZookeeperNodes=1 --parameter KafkaNodes=1 --parameter DataNodes=3 --parameter OpentsdbNodes=1 --parameter PndaFlavor=standard --parameter KeyName=pnda squirrel-land
**ERROR: Property error: : resources.private_subnet.properties.dns_nameservers: : "u"**[u'173.38.200.100']"" is not a list
```

**Comment by jgarnier** [comment]
Hi,

Could you give more details about your configuration file pnda_env_standard.yaml please?

**Comment by 10186937** [comment]
Yeah, the changes of pnda_env_standard.yaml existing on the pnda-heat-templates dir are as follows:

```
 # KEYSTONE CREDENTIALS
  # Uncomment and initialize this snippet's values in regards to your tenant
  keystone_user: 'pnda'
  keystone_password: 'admin'
  keystone_tenant: 'pnda'
  keystone_auth_url: 'http://192.168.1.110/identity_v2_admin'
  keystone_region_name: 'RegionOne'

  # Swift containers and folders
  pnda_apps_container:  'pnda.applications'
  pnda_apps_folder:  'releases'
  pnda_archive_container:  'pnda_archives'
```

Thanks you.

**Comment by jgarnier** [comment]
did you modify name_servers ?

**Comment by 10186937** [comment]
No, I just followed the `README.md` of pnda-heat-templates.
More details:  My os is  centos and the  openstack  env was deployed with devstack.

**Comment by jgarnier** [comment]
ok what is the OpenStack version?

**Comment by 10186937** [comment]
The master branch.

**Comment by jgarnier** [comment]
pnda-heat-template is currently running on OpenStack Kilo so maybe a template issue on newer version

**Comment by trsmith2** [comment]
Specifically this is an issue from Newton and will be fixed in PNDA 3.5.
